### DISCUSSION TITLE
ikun男孩

### DISCUSSION THREAD

**Discussion by lmb1113** [hajimehoshi/ebiten#2900] [Show and tell]
[github](https://github.com/lmb1113/game-go)
![image](https://github.com/hajimehoshi/ebiten/assets/39643887/37662140-ea9f-482a-b1d9-cb5852148b19)

**Comment by looyer** [comment]
Thanks. I already download and build running. It's interesting.
![QQ截图20240228161250](https://github.com/hajimehoshi/ebiten/assets/4188230/ed64d20a-b54f-43a2-8386-3344052ac15f)

**Reply by lmb1113** [reply]
小黑子
